ts/utils/api.ts

```diff
@@ -2,28 +2,10 @@
  * Implements the APIs to interact with the backend.
```

Comment:
Not the file we are using to make the api requests already removed in #130. Look at `ts/api/index.ts`